xclim/sdba/base.py

```diff
@@ -501,7 +507,9 @@ def _decode_cf_coords(ds):
             del ds[crdname].encoding["dtype"]
 
 
-def map_blocks(reduces: Sequence[str] | None = None, **outvars):  # noqa: C901
+def map_blocks(  # noqa: C901
+    reduces: Sequence[str] | None = None, **out_vars
```

Comment:
Is the `outvars` API change documented ?

Reply:
Oh! I hadn't see that change.

However, this is not an API change, is it? `out_vars` is the `kwargs`. It doesn't appear in any calls of `map_blocks` it is only used within the function.

Reply:
Yes, sorry, I think I just changed this for aesthetics/consistency. I didn't think it merited any note since it's a kwargs alias.

Reply:
Right ! Monday...